{"level":"error","message":"Error occurred during plagiarism check Invalid URL","stack":"TypeError: Invalid URL\n    at new URL (node:internal/url:818:25)\n    at dispatchHttpRequest (file:///D:/thakur_hackathon/node_modules/axios/lib/adapters/http.js:232:20)\n    at file:///D:/thakur_hackathon/node_modules/axios/lib/adapters/http.js:152:5\n    at new Promise (<anonymous>)\n    at wrapAsync (file:///D:/thakur_hackathon/node_modules/axios/lib/adapters/http.js:132:10)\n    at http (file:///D:/thakur_hackathon/node_modules/axios/lib/adapters/http.js:170:10)\n    at Axios.dispatchRequest (file:///D:/thakur_hackathon/node_modules/axios/lib/core/dispatchRequest.js:51:10)\n    at Axios._request (file:///D:/thakur_hackathon/node_modules/axios/lib/core/Axios.js:178:33)\n    at Axios.request (file:///D:/thakur_hackathon/node_modules/axios/lib/core/Axios.js:40:25)\n    at Axios.httpMethod [as post] (file:///D:/thakur_hackathon/node_modules/axios/lib/core/Axios.js:217:19)\n    at Axios.request (file:///D:/thakur_hackathon/node_modules/axios/lib/core/Axios.js:45:41)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async checkPlagiarism (file:///D:/thakur_hackathon/controllers/plagiarismController.js:38:29)","timestamp":"2025-02-20T07:37:51.338Z"}
